tests/json/shape_json.py

```diff
@@ -163,7 +163,7 @@ def off_shape_json() -> Dict:
           "type":"dataset",
           "name":"faces",
           "dataset":{
-            "type":"int32",
+            "type":"Integer",
```

Comment:
These type strings were correct before, see filewriter documentation: https://github.com/ess-dmsc/kafka-to-nexus/blob/master/documentation/commands.md